OracleJava/16/Dockerfile

```diff
@@ -0,0 +1,89 @@
+# Copyright (c) 2020,2021 Oracle and/or its affiliates. All rights reserved.
+#
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+#
+# ORACLE DOCKERFILES PROJECT
+# --------------------------
+# This is the Dockerfile for Oracle JDK 16
+#
+# REQUIRED FILES TO BUILD THIS IMAGE
+# ----------------------------------
+#
+# (1) jdk-16.XX_linux-x64_bin.tar.gz
+#     Download from https://www.oracle.com/java/technologies/javase-jdk16-downloads.html
+#
+# HOW TO BUILD THIS IMAGE
+# -----------------------
+# Put all downloaded files in the same directory as this Dockerfile
+# Run:
+#      $ docker build -t oracle/jdk:16 .
+#
+# This command is already scripted in build.sh so you can alternatively run
+#		$ bash build.sh
+#
+# The builder image will be used to uncompress the tar.gz file with the Java Runtime.
+
+FROM oraclelinux:8-slim as builder
```

Comment:
This is just a comment: there's no reason to use `oraclelinux:8-slim` as a builder. If you use `oraclelinux:8` you get the real `dnf`. Doesn't make much difference here, but I thought I'd point it out.